docs/gallery/advanced_io/streaming.py

```diff
@@ -1,98 +1,119 @@
 '''
 .. _streaming:
 
-Streaming from an S3 Bucket
-===========================
+Streaming NWB files
+===================
 
-It is possible to read data directly from an S3 bucket, such as data from the `DANDI Archive
-<https://dandiarchive.org/>`_. This is especially useful for reading small pieces of data
-from a large NWB file stored remotely. In fact, there are two different ways to do this supported by PyNWB.
+You can read specific sections within individual data files directly from remote stores such as the
+`DANDI Archive <https://dandiarchive.org/>`_. This does not require you to download the entire large file just to
+access a small amount of datais especially useful for reading small pieces of data from a large NWB file stored
+remotely. First, you will need to get the location of the file. The code below illustrates how to do this on DANDI
+using the dandi API library.
 
-Method 1: ROS3
-~~~~~~~~~~~~~~
-ROS3 stands for "read only S3" and is a driver created by the HDF5 group that allows HDF5 to read HDF5 files
-stored on s3. Using this method requires that your HDF5 library is installed with the ROS3 driver enabled. This
-is not the default configuration, so you will need to make sure you install the right version of h5py that has this
-advanced configuration enabled. You can install HDF5 with the ROS3 driver from `conda-forge
-<https://conda-forge.org/>`_ using ``conda``. You may first need to uninstall a currently installed version of h5py.
+Getting file location on DANDI
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-'''
+The :py:class:`~dandi.dandiapi.DandiAPIClient` can be used to get the S3 URL of any NWB file stored in the DANDI
+Archive. If you have not already, install the latest release of the ``dandi`` package.
+
+
+.. code-block:: bash
+
+   pip install dandi
+
+Now you can get the url of a particular NWB file using the dandiset ID and the path of that file within the dandiset.
+
+.. code-block:: python
+
+   from dandi.dandiapi import DandiAPIClient
+
+   dandiset_id = '000006'  # ephys dataset from the Svoboda Lab
+   filepath = 'sub-anm372795/sub-anm372795_ses-20170718.nwb'  # 450 kB file
+   with DandiAPIClient() as client:
+       asset = client.get_dandiset(dandiset_id, 'draft').get_asset_by_path(filepath)
+       s3_url = asset.get_content_url(follow_redirects=1, strip_query=True)
+
+
+Streaming Method 1: ROS3
+~~~~~~~~~~~~~~~~~~~~~~~~
+ROS3 is one of the supported methods for reading data from a remote store. ROS3 stands for "read only S3" and is a
+driver created by the HDF5 Group that allows HDF5 to read HDF5 files stored remotely in s3 buckets. Using this method
```

Comment:
HDF5 is just a format... most likely what was meant is libhdf5 library but here it doesn't really need it anyways IMHO, and consistent S3:

```suggestion
driver created by the HDF5 Group that allows to read HDF5 files stored remotely in S3 buckets. Using this method
```